package.json

```diff
@@ -81,7 +81,7 @@
     "typescript": "^4.4.4"
   },
   "dependencies": {
-    "@graphql-typed-document-node/core": "3.1.0",
```

Comment:
This was contributed in https://github.com/zalando-incubator/graphql-jit/pull/139

@ardatan do you recall any special reason why we would have locked the version to `3.1.0`?

Reply:
yeah, ideally it'd just be `^3.1.0` I guess 🙂